project/Dependencies.scala

```diff
@@ -9,7 +9,7 @@ object Dependencies {
     val fs2        = "2.5.0"
     val log4cats   = "1.1.1"
 
-    val lettuce = "6.0.1.RELEASE"
+    val lettuce = "6.1.0.M1"
```

Comment:
Yeah, let's leave this update out of this PR, the rest looks good :+1:

Reply:
@gvolpe One problem is I believe these commands only exist in lettuce on the M1 release. https://github.com/lettuce-io/lettuce-core/blob/f6a43fcd000fa89c79cb2bd770021347db94b329/RELEASE-NOTES.md#L132

Reply:
https://github.com/lettuce-io/lettuce-core/blob/f6a43fcd000fa89c79cb2bd770021347db94b329/RELEASE-NOTES.md#commands

Reply:
Ah that's a bummer. We'll need to leave it on stand-by until the `6.1.0.M1` release then.

Reply:
Yeah, sounds good to me @gvolpe

Reply:
Awesome, gimme a shout when the release comes out, in case I miss it. Usually, @scala-steward takes care of such updates.

Reply:
@calvinbrown085 `6.1.0.RELEASE` is out. In the `series/1.x` (CE3 branch) is already updated but not yet in `master`.